### PR TITLE
Add missing api.PaymentRequestEvent.changePaymentMethod feature

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is a part of a project to add missing interfaces and interface features to BCD that are from an active spec (including WICG specs) and is supported in at least one browser.  This particular PR adds the missing `changePaymentMethod` member of the PaymentRequestEvent API, populating the results using data from the [mdn-bcd-collector](https://mdn-bcd-collector.appspot.com) project (v3.1.10).

Spec: https://w3c.github.io/payment-handler/#changepaymentmethod-method

Tests Used: https://mdn-bcd-collector.appspot.com/tests/api/PaymentRequestEvent/changePaymentMethod

### CODE REVIEW

File: api/PaymentRequestEvent.json [hunk 1]
```diff
@@ -98,6 +98,54 @@
...
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
```
Comment (foolip) [inline api/PaymentRequestEvent.json:124]: Is this really right?

File: api/PaymentRequestEvent.json [hunk 2]
```diff
@@ -98,6 +98,54 @@
...
+            },
+            "samsunginternet_android": {
+              "version_added": "12.0"
+            },
+            "webview_android": {
+              "version_added": false
+            }
+          },
+          "status": {
+            "experimental": false,
```
Comment (foolip) [inline api/PaymentRequestEvent.json:143]: Should be experimental.

**Review by foolip — CHANGES_REQUESTED** [review]

File: api/PaymentRequestEvent.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/PaymentRequestEvent.json:124]: Yeah, Opera doesn't support the payment APIs.

Comment (foolip) [reply api/PaymentRequestEvent.json:124]: Huh, OK :)

**Review by foolip — APPROVED** [review]

**Pull request merged**